failures-summary-and-bottle-result/action.yml

```diff
@@ -1,21 +1,34 @@
-name: Failures summary and bottle result
-description: Output the logs of the bottle generation and a summary of all step failures
+name: '`brew test-bot` result summary'
+description: Output the logs stored at the specified path
 inputs:
   workdir:
     description: Working directory of the result
     required: true
     default: ${{ github.workspace }}
+  result_path:
+    description: Path to the result (relative to workdir)
+    required: false
+    default: bottles/steps_output.txt
+  step_name:
+    description: Heading to use in GitHub step summary
+    required: false
+    default: '`brew test-bot` output'
 
 runs:
   using: composite
   steps:
     - run: |
-        touch ${{ inputs.workdir }}/bottles/steps_output.txt
-        cat ${{ inputs.workdir }}/bottles/steps_output.txt
-        rm ${{ inputs.workdir }}/bottles/steps_output.txt
-      shell: bash
+        if [[ -f "$GITHUB_STEP_SUMMARY" ]] &&
```

Comment:
Not 100% sure if `$GITHUB_STEP_SUMMARY` is inherited from the calling workflow, or if I need to pass this variable. If I do need to pass it, I'm not sure how, since the Bash variable is likely not to be defined where I do `with:` in the calling workflow.